velox/functions/lib/aggregates/MinMaxAggregateBase.cpp

```diff
@@ -427,7 +427,6 @@ class MinMaxAggregateBase : public exec::Aggregate {
       const VectorPtr& arg,
       TCompareTest compareTest) {
     DecodedVector decoded(*arg, rows, true);
-    auto indices = decoded.indices();
```

Comment:
There are other functions in this file that follow a similar pattern. Should we apply the same modification to them as well?
For example: `doUpdate`.